### ISSUE TITLE
Cached graph is saved to JSONField as a string, should be dictionary/object

### ISSUE THREAD

**Issue by robgaston** [archesproject/arches#8172]
We save a serialized version of the graph models to `graph_publications.serialized_graph`, which is a JSONField.  Currently, this is stored as a string, which causes some issues with how we handle the serialized graph (see the add resource page).  It also would be more useful/usable in the database if stored as an object.

When we assign this column, it should be using a dictionary so that we store a proper JSON object.

**Comment by robgaston** [comment]
One thing that this implies is a new migration to update existing publications to use objects rather than strings.

**Comment by chiatt** [comment]
Interesting that Postgres even allows that to happen

**Comment by robgaston** [comment]
@chiatt yeah, you might not expect it, but valid JSON values in PostgreSQL valid JSON values can be any of the following: object, array, string, number, boolean, or null
